### ISSUE TITLE
Log entry search API

### ISSUE THREAD

**Issue by heuels** [briskhome/briskhome#31]
In parallel with developing **core.utils** component that holds APIs for interacting with devices, services, sensors and readings an addition to **core.log** component should be developed that will provide a simple API for accessing log records.

**Comment by heuels** [comment]
Since support for logging to the database is dropped this feature is no longer relevant.
